outgoingra/src/main/java/no/bankid/outgoing/ra/AuthenticationBodyDTO.java

```diff
@@ -3,18 +3,32 @@
 import io.swagger.v3.oas.annotations.media.Schema;
 
 @Schema(description = "Minimal body for authenticating and authorizing a request")
-
 public class AuthenticationBodyDTO {
-    @Schema(description = "Name of client sending the request",
-            example = "vipps-bapp-client")
+    @Schema(
+            description = "Name of client sending the request",
+            example = "vipps-bapp-client",
```

Comment:
```suggestion
            example = "bankid-bapp-client",
```

Reply:
I think we are still using `vipps-bapp-client` in the code, if I am not mistaken.

Reply:
Yes we do not want to change that. `vipps-bapp-client` is the client name for RA-Light (the Preprod RA)